src/components/SQLResultTable/SQLResultsTable.tsx

```diff
@@ -2,13 +2,13 @@ import React from 'react';
 import { Table, Tabs } from 'antd';
```

Comment:
The component is `SQLResultsTable` but the folder is `SQLResultTable`, maybe rename the folder to be in line?